dataflow/src/main/java/org/checkerframework/dataflow/util/DataflowPerformer.java

```diff
@@ -0,0 +1,60 @@
+package org.checkerframework.dataflow.util;
+
+import org.checkerframework.dataflow.analysis.Analysis;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizeLauncher;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * It's basically a class that provides utility for running some dataflow analysis tests and
+ * playgrounds.
+ */
+public class DataflowPerformer {
+
+    /**
+     * The performTest method performs the tests for some dataflow analysis.
+     *
+     * @param analysis instance of forward or backward analysis from specific dataflow test case.
+     */
+    @SuppressWarnings("CatchAndPrintStackTrace") // we want to use e.printStackTrace here.
+    public static void performTest(Analysis<?, ?, ?> analysis) {
+        String inputFile = "Test.java";
+        String method = "test";
+        String clazz = "Test";
+        String outputFile = "Out.txt";
+
+        CFGVisualizeLauncher cfgVisualizeLauncher = new CFGVisualizeLauncher();
+        Map<String, Object> res =
+                cfgVisualizeLauncher.generateStringOfCFG(inputFile, method, clazz, true, analysis);
```

Comment:
Can you think whether it would be nicer to move these two new functions into dataflow/src/main/java/org/checkerframework/dataflow/cfg/visualize/CFGVisualizeLauncher.java
as static functions that make it easier to use?

Reply:
Sure, done.